### ISSUE TITLE
Client error / Could not fetch changelog for Paradox Alarm Interface 3.4.1 since 2024.04.4

### ISSUE THREAD

**Issue by MacrosorcH** [home-assistant/supervisor#5053]
### Describe the issue you are experiencing

Since upgrading HA Supervisor from  2024.04.0 to 2024.04.4, HA Core started logging the following errors related to Paradox Alarm Interface (PAI 3.4.1) every 5 minutes:

2024-05-01 13:26:41.407 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/e92a3241_paradox_alarm_interface_3.x/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/e92a3241_paradox_alarm_interface_3.x/changelog')
2024-05-01 13:26:41.408 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for e92a3241_paradox_alarm_interface_3.x:
2024-05-01 13:31:41.683 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/e92a3241_paradox_alarm_interface_3.x/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/e92a3241_paradox_alarm_interface_3.x/changelog')
2024-05-01 13:31:41.684 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for e92a3241_paradox_alarm_interface_3.x:
2024-05-01 13:36:41.682 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/e92a3241_paradox_alarm_interface_3.x/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/e92a3241_paradox_alarm_interface_3.x/changelog')
2024-05-01 13:36:41.683 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for e92a3241_paradox_alarm_interface_3.x:
2024-05-01 13:41:41.681 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/e92a3241_paradox_alarm_interface_3.x/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/e92a3241_paradox_alarm_interface_3.x/changelog')
2024-05-01 13:41:41.682 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for e92a3241_paradox_alarm_interface_3.x:
2024-05-01 13:46:41.685 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/e92a3241_paradox_alarm_interface_3.x/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/e92a3241_paradox_alarm_interface_3.x/changelog')
2024-05-01 13:46:41.686 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for e92a3241_paradox_alarm_interface_3.x:

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### Steps to reproduce the issue

1. Until Supervisor version 2024.04.0 with PAI 3.4.1 no errors
2. Upgrade to Supervisor version 2024.04.4
3. Check logs and you'll find errors every 5 minutes

### Anything in the Supervisor logs that might be useful for us?

```txt
No errors in Supervisor log - the errors are in HA Core log
```

## System Information

version | core-2024.5.0b4
-- | --
installation_type | Home Assistant OS
dev | false
hassio | true
docker | true
user | root
virtualenv | false
python_version | 3.12.2
os_name | Linux
os_version | 6.1.73-haos-raspi
arch | aarch64
timezone | Europe/Rome
config_dir | /config

<details><summary>Home Assistant Community Store</summary>

GitHub API | ok
-- | --
GitHub Content | ok
GitHub Web | ok
GitHub API Calls Remaining | 4824
Installed Version | 1.34.0
Stage | running
Available Repositories | 1400
Downloaded Repositories | 16

</details>

<details><summary>Home Assistant Cloud</summary>

logged_in | true
-- | --
subscription_expiration | 9 luglio 2024 alle ore 02:00
relayer_connected | true
relayer_region | eu-central-1
remote_enabled | false
remote_connected | false
alexa_enabled | false
google_enabled | false
remote_server | eu-central-1-6.ui.nabu.casa
certificate_status | ready
instance_id | 24d993679dc748ac9dccd29bbf807bc5
can_reach_cert_server | ok
can_reach_cloud_auth | ok
can_reach_cloud | ok

</details>

<details><summary>Home Assistant Supervisor</summary>

host_os | Home Assistant OS 12.2
-- | --
update_channel | stable
supervisor_version | supervisor-2024.04.4
agent_version | 1.6.0
docker_version | 25.0.5
disk_total | 116.5 GB
disk_used | 15.9 GB
healthy | true
supported | true
board | rpi4-64
supervisor_api | ok
version_api | ok
installed_addons | Duck DNS (1.17.0), File editor (5.8.0), Home Assistant Google Drive Backup (0.112.1), NGINX Home Assistant SSL proxy (3.9.0), Piper (1.5.0), ESPHome (2024.4.2), openWakeWord (1.10.0), Terminal & SSH (9.13.0), Paradox Alarm Interface (3.4.1), Mosquitto broker (6.4.0), Samba share (12.3.1)

</details>

<details><summary>Dashboards</summary>

dashboards | 2
-- | --
resources | 13
views | 30
mode | storage

</details>

<details><summary>Recorder</summary>

oldest_recorder_run | 23 aprile 2024 alle ore 16:49
-- | --
current_recorder_run | 1 maggio 2024 alle ore 14:09
estimated_db_size | 1354.71 MiB
database_engine | sqlite
database_version | 3.44.2

</details>

**Comment by TNTLarsn** [comment]
It's on other Addons too:

```logs
Client error on /addons/40817795_vscode-remote/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/40817795_vscode-remote/changelog')

Client error on /addons/a4f7bb3e_leaf2mqtt/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/a4f7bb3e_leaf2mqtt/changelog')
```

**Comment by Chaoscontrol** [comment]
Same here with other addons:
```
2024-05-05 10:10:15.140 ERROR (MainThread) [homeassistant.components.hassio.handler] Client error on /addons/eb27a660_hyperhdr/changelog request 0, message='Attempt to decode JSON with unexpected mimetype: application/octet-stream', url=URL('http://172.30.32.2/addons/eb27a660_hyperhdr/changelog')
2024-05-05 10:10:15.140 WARNING (MainThread) [homeassistant.components.hassio.data] Could not fetch changelog for eb27a660_hyperhdr:
```

**Comment by Destroyer061090** [comment]
same issues

**Comment by SeeThisIsMe** [comment]
Yeah this is a problem, I'm getting it with different addons, thousands of logs per day.

**Comment by Destroyer061090** [comment]
if you go to addons and looking for the logs do you have the same error?
![image](https://github.com/home-assistant/supervisor/assets/126196958/5c6ed275-aea6-4431-ac3e-17a05f01b1d4)

**Comment by agners** [comment]
Fixed with #5064 in Supervisor 2024.05.0 (currently on beta channel).
